src/notebooks/controllers/controllerPreferredService.ts

```diff
@@ -422,25 +420,6 @@ export class ControllerPreferredService implements IControllerPreferredService,
         serverId: string | undefined
     ): Promise<KernelConnectionMetadata | undefined> {
         const uri = notebook.uri;
-        if (KernelPickerType.useNewKernelPicker) {
-            // With the new kernel picker always find exact matches when updating the affinity.
-            // Attempt to clean up https://github.com/microsoft/vscode-jupyter/issues/11914
-            // Here we actually need an exact match instead of close matches,
-            // For the new kernel picker we will always try to find an exact match, if not found, then we
-            // do not show anything in the VS Codes kernel picker.
-            const preferredService = new PreferredKernelConnectionService();
-            try {
-                if (serverId) {
-                    return preferredService.findExactRemoteKernelConnection(notebook, cancelToken);
```

Comment:
No need to match exactly, fuzzy is good enough (this way we still have a controller dipslayed in vs code kernel picker)
Removing code path for the new kernel picker to make it behave the same as old (stable) kernel picker